src/css/editor.css.js

```diff
@@ -115,29 +115,34 @@ module.exports = `
     background-repeat: no-repeat;
 }
 
-.ace_gutter-cell_svg-icons .ace_icon_svg{
+.ace_gutter-cell_svg-icons .ace_icon_svg {
     margin-left: -14px;
     float: left;
 }
 
-.ace_gutter-cell.ace_error, .ace_icon.ace_error {
+.ace_gutter-cell .ace_icon {
+    margin-left: -18px;
+    float: left;
+}
+
+.ace_gutter-cell.ace_error, .ace_gutter-tooltip .ace_icon.ace_error {
```

Comment:
I think this change will break cloud9, because there we use custom gutter cell backgrounds for errors and breakpoints. With this both will be visible. 

If the goal is simply to have a node for getting client rect, maybe we can use client rect from element and offset by `editor.renderer.$gutterLayer.$padding.left`?

Reply:
Let's try to not break c9 🙂 could you explain how this change could affect c9? 

I assume they use selectors like `.ace_gutter-cell.ace_error` for their custom backgrounds and nothing changes for these selectors. The only change here is that the background set for `.ace_icon.ace_error` is now scoped to `.ace_gutter-tooltip` (to avoid the background image showing up twice).

Reply:
My bad, i thought this changes ace to set background on `ace_icon` instead of `ace_gutter-cell`, which would break the css override in cloud9.

I still think using the value from padding instead of adding a completely empty element would be better, but this may be ok too.